modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,147 @@
+:_mod-docs-content-type: PROCEDURE
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running builds using SharedSecret objects
+
+You can use a `SharedSecret` object to securely access the entitlement keys of a cluster in builds.
+
+The `SharedSecret` object allows you to share and synchronize secrets across namespaces.
+
+[IMPORTANT]
+====
+The Shared Resource CSI Driver feature is now generally available in link:https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1[{builds-v2title} 1.1]. This feature is now deprecated in {product-title}. To use this feature, ensure you are using {builds-v2title} 1.1 or a more recent version.
+====
+
+.Prerequisites
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

Comment:
Do we still need this? As we are announcing the feature as GA, and it's available with builds, I don't think there's any need to enable `TechPreviewNoUpgrade` now.